notebooks/2019-11-12_Calendars_and_monthly_data.ipynb

```diff
@@ -0,0 +1,576 @@
+   "source": [
+    "We'll read the files and keep all the data in a list called `var_list`. We don't need anything fancier than a list because we are keeping all the metadata we need to identify our data with the DataArray itself."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<xarray.DataArray 'pr' (time: 300, lat: 145, lon: 192)>\n",
+      "[8352000 values with dtype=float32]\n",
+      "Coordinates:\n",
+      "  * time      (time) object 2274-12-16 00:00:00 ... 2299-11-16 00:00:00\n",
+      "  * lat       (lat) float64 -90.0 -88.75 -87.5 -86.25 ... 86.25 87.5 88.75 90.0\n",
+      "  * lon       (lon) float64 0.0 1.875 3.75 5.625 7.5 ... 352.5 354.4 356.2 358.1\n",
+      "    model_id  <U10 'HadGEM2-ES'\n",
+      "Attributes:\n",
+      "    standard_name:     precipitation_flux\n",
+      "    long_name:         Precipitation\n",
+      "    comment:           at surface; includes both liquid and solid phases from...\n",
+      "    units:             kg m-2 s-1\n",
+      "    original_name:     mo: m01s05i216\n",
+      "    history:           2011-11-24T09:57:29 out-of-bounds adjustments: (-1e-07...\n",
+      "    cell_methods:      time: mean\n",
+      "    cell_measures:     area: areacella\n",
+      "    associated_files:  baseURL: http://cmip-pcmdi.llnl.gov/CMIP5/dataLocation... \n",
+      " <xarray.DataArray 'pr' (time: 2412, lat: 64, lon: 128)>\n",
+      "[19759104 values with dtype=float32]\n",
+      "Coordinates:\n",
+      "  * time      (time) object 2100-01-16 12:00:00 ... 2300-12-16 12:00:00\n",
+      "  * lat       (lat) float64 -87.86 -85.1 -82.31 -79.53 ... 82.31 85.1 87.86\n",
+      "  * lon       (lon) float64 0.0 2.812 5.625 8.438 ... 348.8 351.6 354.4 357.2\n",
+      "    model_id  <U10 'bcc-csm1-1'\n",
+      "Attributes:\n",
+      "    standard_name:     precipitation_flux\n",
+      "    long_name:         Precipitation\n",
+      "    comment:           at surface; includes both liquid and solid phases from...\n",
+      "    units:             kg m-2 s-1\n",
+      "    original_name:     PRECC+PRECL\n",
+      "    cell_methods:      time: mean (interval: 20 mintues)\n",
+      "    cell_measures:     area: areacella\n",
+      "    associated_files:  baseURL: http://cmip-pcmdi.llnl.gov/CMIP5/dataLocation...\n"
+     ]
+    }
+   ],
+   "source": [
+    "var_list = []\n",
+    "for ffile in filename:\n",
+    "    ds = xr.open_dataset(ffile)\n",
+    "    # Let's grab the model name from the attributes and add as a coordinate. \n",
+    "    # That will help with doing averages across the mode dimension. \n",
+    "    # Additionally, the model_id information is kept when looking at the variable itself \n",
+    "    # when using a coordinate while the global attribute is lost.\n",
+    "    ds = ds.assign_coords({'model_id':ds.model_id})\n",
```

Comment:
Not worth changing anything, but in the future it may be clearer to use a dict for stuff like this, where you want data attached to a name